packages/core/src/Modules/Cashier/Components/on-ramp-provider-card.jsx

```diff
@@ -1,43 +1,101 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Button, Icon } from '@deriv/components';
+import { Button, Icon, MobileWrapper, DesktopWrapper } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 
-const OnRampProviderCard = ({ provider, setSelectedProvider }) => (
-    <div className='on-ramp__provider'>
-        <div className='on-ramp__provider-logo'>
-            <Icon icon={provider.icon} width={118} height={28} />
-        </div>
-        <h2 className='on-ramp__provider-name'>{provider.name}</h2>
-        <div className='on-ramp__provider-description'>{provider.description}</div>
-        <a href={provider.website} className='on-ramp__provider-website link' target='_blank' rel='noopener noreferrer'>
-            {provider.website}
-        </a>
-        {provider.payment_icons.length && (
+const ProviderIcons = ({ providers, is_dark_mode_on }) => (
+    <div className='on-ramp__provider-patment-icons-box'>
+        {providers?.length && (
             <div className='on-ramp__provider-payment-icons'>
-                {provider.payment_icons.map((payment_icon, idx) => (
-                    <Icon key={idx} size={40} icon={payment_icon} />
+                {providers.map((provider, idx) => (
+                    <Icon
+                        key={idx}
+                        size={40}
+                        icon={provider.icon}
+                        custom_color={
+                            provider.custom_color
+                                ? is_dark_mode_on
+                                    ? provider.dark_mode_fill
+                                    : provider.light_mode_fill
+                                : ''
```

Comment:
Try to avoid nesting ternary statements (a warning should be emitted when running tests), since you're inside a `map` you can define a variable above line 12 instead